### PR TITLE
Updating work process

### PR DESCRIPTION
**Pull request by dmurko**
Ref https://github.com/niteoweb/operations/issues/660

### CODE REVIEW

File: 2_Operations/work-process.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ For each new sprint, the Scrum master will create an operations sprint release t
 
 ## Scrum
 
-Our work process is based on [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)). Read in detail about [our Scrum process](scrum.md).
+Our work process is based on [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)). Read in detail about [our Scrum process](scrum.md) and writing [User Stories](user-stories.md).
 
  * Sprint length is two weeks.
  * Story points commitment: 10 for each full-time member on the sprint. The total team commitment has at least 25% reserved for bugfixes and cleanup.
```
Comment (marbeann) [inline 2_Operations/work-process.md]: * bug fixes

File: 2_Operations/work-process.md [hunk 2]
```diff
@@ -31,13 +31,15 @@ Primarily we look over the quarterly goals to determine how we are achieving the
...
 The main purpose of the alignment meeting is to adjust our compasses so that we will select the correct user stories in the sprint planning meeting coming the next day. We know we are paddling hard, but are we all paddling in the right direction?
 
-
 ## Commits and Pull Requests
 
 While working on user stories we often need to change code or text that we keep in GitHub. These changes are done with [git commits](https://help.github.com/articles/github-glossary/#commit) and are presented to others for review as [Pull Requests](https://help.github.com/articles/about-pull-requests/).
 
 To ensure that your commits and Pull Requests contain everything necessary for a successful review, read through the [Git](/3_Development/git.md) guide for the best practices.
 
+After issue/user story/pull request is getting close to being completed people should go on zoom and discuss what else needs to be fixed. We don't want to see long PR discussions on GitHub. The **author** of the issue/user story/pull request should ping people on slack and get them on zoom to polish things.
```
Comment (marbeann) [inline 2_Operations/work-process.md]: >After issue/user story/pull request is getting close to being completed people should go on zoom and discuss what else needs to be fixed.

1. Sentence is off. Maybe change `After` to `When an`.

Or `Prior to closing an issue/user story/pull request, people should go on zoom and discuss what else needs to be fixed.`

2. Capitalize `Zoom` and `Slack`.

File: 2_Operations/work-process.md [hunk 3]
```diff
@@ -1,65 +1,52 @@
...
- * The Tuesday, two weeks later, marks the end of the sprint with a [sprint retrospective](scrum.md#Sprint_Retrospective).
+ * A Wednesday is the start of a sprint with sprint planning.
+ * The Tuesday, two weeks later, marks the end of the sprint with a sprint retrospective.
  * Then Wednesday, the next day, is usually the start of the next sprint.
 
+The last Monday morning of the sprint everyone should open up the Scrum board and ask themselves: "How can I help close whatever is still opened?". Repeat the same after lunch and on Tuesday morning.
 
-## Creating issues and user stories
-> *Main document: [Guide for user stories](user-stories.md)*
+For each new sprint, the Scrum master will create an operations sprint release that is used to store the user story screencast demos. The template for this release is copied from [RELEASE_TEMPLATE.md](https://github.com/niteoweb/operations/raw/master/.github/RELEASE_TEMPLATE.md) in the operations repo.
```
Comment (zupo) [inline 2_Operations/work-process.md]: Do we have demos from both ebn and woocart sprints in the same release? If not, the releases should be in their respective repositories.

File: 2_Operations/work-process.md [hunk 4]
```diff
@@ -1,65 +1,52 @@
...
+## Quarterly Planning
 
-> *Main article: [Git guide](../apps-we-use/git.md)*
+TODO
 
-While working on user stories we often need to change code or text that we keep in GitHub. These changes are done with [git commits](https://help.github.com/articles/github-glossary/#commit) and are presented to others for review as [Pull Requests](https://help.github.com/articles/about-pull-requests/).
+## Alignment Meeting
 
-To ensure that your commits and Pull Requests contain everything necessary for a successful review, read through the [Git](../apps-we-use/git.md) guide for the best practices.
+15 minutes after the sprint retrospective we hold another meeting. Attendance on this one is optional, only those interested in the general direction the company is taking normally attend. That said, every Nitean is welcome to participate.
```
Comment (zupo) [inline 2_Operations/work-process.md]: Since we now have two retrospectives, this should probably be scheduled at 12 CE(S)T.

Comment (zupo) [reply 2_Operations/work-process.md]: Alternatively, we could have one alignment for EBN and one for WooCart, each fro their own part of the OKRs. 

I'd still to do it together though ... maybe we need to shuffle people between projects if one is falling behind ...

Comment (dmurko) [reply 2_Operations/work-process.md]: This change will be implemented after 26th September.

File: 2_Operations/work-process.md (hunk 3 above)
Comment (dmurko) [reply 2_Operations/work-process.md]: In their respective repositories.

**Pull request merged**